content/post/GSOC_AMA_report.md

```diff
@@ -105,6 +105,13 @@ Every bit of this, couldn't be completed or even imagined without the help and d
 <blockquote class="twitter-tweet" data-lang="en" align=center><p lang="en" dir="ltr">A special thanks to <a href="https://twitter.com/vipulgupta2048?ref_src=twsrc%5Etfw">@vipulgupta2048</a> for mentoring members and conducting the AMA Sessions on GSoC . <br>A heartful thank you to the ALiAS Volunteer team especially Raghav, Siddhant, Akshat, Shivam, Sahil and Kaushlendra and every other soul who helped along the way. <a href="https://t.co/Ich2qlRNHe">pic.twitter.com/Ich2qlRNHe</a></p>&mdash; ALiAS (@AsetALiAS) <a href="https://twitter.com/AsetALiAS/status/1125603066642780160?ref_src=twsrc%5Etfw">May 7, 2019</a></blockquote>
 <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
 
+## [Update] ALiAS hits its mark in Alibaba Summer of Code 2019
+
+[Sahil Jha](https://twitter.com/sjha2048), a sophomore from the Computer Science Dept. of Amity University, and a long-standing member of our community was just accepted into [Alibaba Summer of code](https://developer.aliyun.com/summerofcode2019) and we are so happy. Join us into congratulating him.
```

Comment:
```suggestion
We are so happy to announce that [Sahil Jha](https://twitter.com/sjha2048), a sophomore from the Computer Science Dept. of Amity University, and a long-standing member of our community was just accepted into [Alibaba Summer of code](https://developer.aliyun.com/summerofcode2019). Join us into congratulating him.
```

Reply:
@vipulgupta2048 Ok then, commit it and merge 
Checked again, everything looks good